lib/slack/bot.ex

```diff
@@ -87,11 +87,11 @@ defmodule Slack.Bot do
       token: token,
       me: rtm.self,
       team: rtm.team,
-      bots: rtm_list_to_map(rtm.bots),
-      channels: rtm_list_to_map(rtm.channels),
-      groups: rtm_list_to_map(rtm.groups),
-      users: rtm_list_to_map(rtm.users),
-      ims: rtm_list_to_map(rtm.ims)
+      bots: rtm_list_to_map(Slack.Web.Bots.info(%{token: token}) |> Map.get("bot")),
```

Comment:
This would kick off 5 different requests at once which I don't think is ideal. Is there no equivalent API to `rtm.start` to bootstrap the client?

Reply:
I can't link to it directly, but https://api.slack.com/rtm has `rtm.connect` vs `rtm.start`. It looks like both are still valid.

I'd be okay with reverting the change to use `rtm.connect` meaning the library will continue to only support `rtm.start` until someone takes the time to add `rtm.connect` support (assuming anyone needs it over `rtm.start`).

That being said, it does look like this approach of making multiple requests is how Slack recommends you use `rtm.connect`.

Reply:
> That being said, it does look like this approach of making multiple requests is how Slack recommends you use rtm.connect.

That, and anyone using the Enterprise Grid with `rtm.start` will not work as expected if we stick with the old solution

Reply:
That is true. I'm not super concerned about supporting it but if we want to move forward with this and we get support at the same time I'm all for it 👍